tests/test_sipApproximation.py

```diff
@@ -577,8 +584,8 @@ def run(original, width, height, order, spacing=20, skyTol=1.0e-4, pixTol=1.0e-8
             # Not comparing to absolute round-trip, but relative to the
             # original WCS round-trip. This is an important distinction: we're
             # recreating the original WCS, along with all its flaws.
-            roundTrip = np.array([wcs.skyToPixel(wcs.pixelToSky(pp)) for pp in points])
-            roundTrip -= np.array([original.skyToPixel(original.pixelToSky(pp)) for pp in points])
+            roundTrip = np.array(wcs.skyToPixelArray(*wcs_sky))
```

Comment:
Do we need to convert this to an array?  Oh, it's because we get an x, y tuple; I don't know if `vstack` would be clearer.

Reply:
Right, we get a tuple of two ndarray back so I convert them again. I can change to vstack.

Reply:
I switched to vstack.